App.tsx

```diff
@@ -33,7 +35,7 @@ const App = () => {
 
   async function configureSMKitUI(){
     try{
-      var res = await configure("YOUR_AUTH_KEY");
+      var res = await configure("public_live_7@4+Ij4avHqIDErk");
```

Comment:
@SencyNetanel to avoid pushing sensitive data to this repo we need to delete this branch, then locally remove this part from your commits and push to a new branch